macros/cross-adapter-modeling/base/segment_web_page_views.sql

```diff
@@ -7,7 +7,50 @@
 
 {% macro default__segment_web_page_views() %}
 
-{{ exceptions.raise_compiler_error("macro segment_web_page_views not implemented for this adapter") }}
+with source as (
+
+    select * from {{var('segment_page_views_table')}}
+    
+),
+
+renamed as (
+
+    select
+    
+        id as page_view_id,
+        anonymous_id,
+        user_id,
+        
+        received_at as received_at_tstamp,
+        sent_at as sent_at_tstamp,        
+        timestamp as tstamp,
+
+        url as page_url,
+        ltrim(split_part(split_part(url, '.com', 1), '//',2),'www.')::varchar 
```

Comment:
I don't think this actually works, right? It would only handle the .com TLD, and not the infinite number of other possible TLDs. We need this package to work with any TLD...typically a majority of traffic is from .com but I would guess that about 1/4 of traffic to most sites would come from other TLDs.

Reply:
Also:
- I don't believe you need to strip out `www.` --that's a part of the host section of the URL. 
- my suggestion on how to solve for all TLDs would be to split on either the first `/` or `?` after the initial `//`. Test it with a bunch of stuff.